recipes/icu/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+cd source
+chmod +x configure install-sh
+
+if [ "$(uname)" == "Linux" ]; then
+    ./configure --prefix="$PREFIX"
```

Comment:
No bit-specification.  Relevant docs: http://userguide.icu-project.org/icufaq#TOC-How-do-I-get-32--or-64-bit-versions-of-the-ICU-libraries-

As for disable-samples and enable-static, I can't find any docs with google.  I assume enable-static does what you expect - but not sure if this also disables shared library building (sometimes it does).  Many examples around the web have --disable-* flags (https://git.busybox.net/buildroot/commit/?id=5f6baf89d487c6aaf5fb3684c48032a3460e2b45) but I haven't found any docs saying what they do, precisely.